ansible_rulebook/app.py

```diff
@@ -35,6 +35,12 @@
     send_event_log_to_websocket,
 )
 
+
+class NullQueue:
+    async def put(self, _data):
```

Comment:
Should it override `put_nowait` method as well?

Reply:
@Alex-Izquierdo I don't see any references in the code for event_log.put_nowait.